codebuild/spec/buildspec_general_batch.yml

```diff
@@ -0,0 +1,116 @@
+---
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may not use
+# this file except in compliance with the License. A copy of the License is
+# located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied. See the License for the specific language governing permissions and
+# limitations under the License.
+version: 0.2
+
+batch:
+  build-list:
+    # Saw
+    - identifier: s2nSawBike
+      buildspec: codebuild/spec/buildspec_ubuntu.yml
+      env:
+        privileged-mode: true
+        compute-type: BUILD_GENERAL1_2XLARGE
```

Comment:
same question, how did you arrive at these compute types?

Reply:
Historical trial and error.  SAW reliably fails on instances w/o enough RAM, but the current job is using a 2XL [here](https://github.com/awslabs/s2n/blob/main/codebuild/codebuild.config#L77)